workinspector/src/main/java/org/koitharu/workinspector/WorkInspectorActivity.kt

```diff
@@ -2,16 +2,32 @@ package org.koitharu.workinspector
 
 import android.os.Bundle
 import android.view.MenuItem
+import android.view.ViewGroup.MarginLayoutParams
+import androidx.activity.enableEdgeToEdge
 import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.ViewCompat
+import androidx.core.view.WindowInsetsCompat
+import androidx.core.view.updateLayoutParams
 import org.koitharu.workinspector.databinding.WiActivityWorkInspectorBinding
 
 public class WorkInspectorActivity : AppCompatActivity() {
     private lateinit var binding: WiActivityWorkInspectorBinding
 
     override fun onCreate(savedInstanceState: Bundle?) {
+        enableEdgeToEdge()
         super.onCreate(savedInstanceState)
         binding = WiActivityWorkInspectorBinding.inflate(layoutInflater)
         setContentView(binding.root)
+        ViewCompat.setOnApplyWindowInsetsListener(binding.appbar) { v, windowInsets ->
+            val insets = windowInsets.getInsets(WindowInsetsCompat.Type.systemBars())
+            v.updateLayoutParams<MarginLayoutParams> {
+                leftMargin = insets.left
+                bottomMargin = insets.bottom
```

Comment:
Are you sure it is a good idea to add a `bottomMargin` to the appBar?

Reply:
Hm...That is a good point 🤔 Probably not.
(It's been a while since I've done views so I'm not as well versed in view insets 😅 )